docker/env.yaml

```diff
@@ -4,7 +4,7 @@ channels:
 dependencies:
   - python=3.8.10
```

Comment:
Why have we pinned to this specific version of Python 3.8?

It's up to 3.8.17 now, so .10 is quite old.

But, also probably not relevant if we jump up to 3.10 or 3.11. #266